Formula/libfreenect.rb

```diff
@@ -13,20 +13,13 @@ class Libfreenect < Formula
     sha256 "f714532e1b21365063746846544a340dac70cf0c5cc877a207dd17284ee100b7" => :mavericks
   end
 
-  option :universal
-
   depends_on "cmake" => :build
   depends_on "libusb"
 
   def install
     args = std_cmake_args
```

Comment:
can just move std_cmake_args down to the system "cmake" line, with the -DBUILD_OPENNI2_DRIVER=ON inline